src/terraform/templates/exocom.tf

```diff
@@ -27,10 +31,8 @@ module "exocom_service" {
   docker_image          = "{{{dockerImage}}}"
   env                   = "production"
   environment_variables = {
-    ROLE = "exocom"
-    SERVICE_ROUTES = <<EOF
-{{{serviceRoutes}}}
-EOF
+    ROLE           = "exocom"
+    SERVICE_ROUTES = "${var.exocom_service_routes}"
```

Comment:
thoughts on updating this to be `environment_variables = "${var.exocom_env_vars}"`? I don't think the role is needed either